src/py_pecg/Subfunction_Module.py

```diff
@@ -0,0 +1,332 @@
+import numpy as np
```

Comment:
According to the [PEP8](https://peps.python.org/pep-0008/) module names in python are lower case with underscores, e.g. `subfunction_module`.

As for the name itself, "subfunction" is sometimes used to describe a nested function. Let's try to find a name capturing the common functionality of the functions in the module... Are they all related to extrema? anything else? If there's absolutely no common topic, we can call the module `utils.py` or something